srcpkgs/devour/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'devour'
+pkgname=devour
+version=11.0
+revision=1
+archs="x86_64* i686*"
+hostmakedepends="libX11-devel"
+depends="libX11"
+short_desc="X11 window swallower"
+maintainer="shmalebx9 <lagrancec477@gmail.com>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/salman-abedin/devour"
+distfiles="https://github.com/salman-abedin/devour/archive/v${version}.tar.gz"
+checksum=e537c614cdf3e7376db1a94f1665d1d27f128d4927f4a0d1cce26ed8b910a217
+
+do_build() {
+	make
+}
+
+do_install() {
+	vbin devour
+}
```

Comment:
You can probably use the `gnu-makefile` build style instead of defining the functions.